subsys/bluetooth/host/classic/sdp.c

```diff
@@ -998,6 +999,11 @@ static uint16_t get_att_search_list(struct net_buf *buf, uint32_t *filter,
 	size = data_elem.data_size;
 
 	while (size) {
+		if (filter_count <= *num_filters) {
```

Comment:
I'd swap the right/left sides - it's more intuitive to have the constant on the right side of the comparison.

Reply:
Updated